### DISCUSSION TITLE
Read me file after right click on getting started button. Please help

### DISCUSSION THREAD

**Discussion by dbaTripleArtStduios** [skills/.github#535] [Q&A]
Ok started my new repository after the Getting started button and I have waited the 20 seconds but I just don't understand enough of this system to understand how to even get back to that introduction to GitHub, Please, anyone that can help.

**Comment by LuckyLuc96** [comment]
I'm having the same issue right now. I think the website may be slow to make that 1st update and it looks like in your case it took an hour for Github to push the readme from step 0 to step 1. I'm hoping mine will complete it's update to step 1 by the time 1 hour hits!
